cmd/bucket-policy-parser.go

```diff
@@ -26,9 +26,12 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/minio/minio-go/pkg/policy"
 	"github.com/minio/minio-go/pkg/set"
 )
 
+var sentinelBucketPolicy = policy.BucketAccessPolicy{}
```

Comment:
Perhaps `emptyBucketPolicy` is a more straightforward name for this? Sentinels are usually used in the context of sorting as a sort of zero/border value.

Reply:
I can do that @donatello - i used sentinel in context of default value.

Reply:
@donatello please review again.